public/index.html

```diff
@@ -12,22 +12,33 @@
     <link href="/css/main.css" rel="stylesheet" />
 </head>
 <body>
-    <header class="navbar navbar-inverse navbar-fixed-top">
-        <div id="cross-site" class="nav-tabs">
+    <header class="navbar navbar-inverse navbar-static-top">
+        <div id="cross-site" class="nav-tabs navbar-collapse collapse">
             <div class="container" role="navigation">
                 <ul class="nav nav-tabs">
                     <li role="presentation"><a href="https://www.histwest.org.au/" class="cross-site-link">Website</a></li>
                     <li role="presentation"><a href="https://shop.histwest.org.au/" class="cross-site-link">Shop</a></li>
-                    <li role="presentation" class="active"><a href="https://research.histwest.org.au/" class="cross-site-link">Research</a></li>
+                    <li role="presentation" class="active"><a href="https://research.histwest.org.au/" class="cross-site-link">Collections</a></li>
                 </ul>
             </div>
         </div>
         <div class="container">
-            <a href="/"><img src="/images/logo-small.png" alt="Royal Western Australian Historical Society" /></a>
+            <a href="/"><img src="/images/rwahs-logo_web.png" alt="Royal Western Australian Historical Society" class="logo" /></a>
+        </div>
+        <div class="container">
+            <div class="nav-main-menu">
+                <ul class="menu nav navbar-nav">
+                    <li><a href="/">Home</a></li>
+                    <li><a href="/library/search/">Library</a></li>
+                    <li><a href="/photographs/search/">Photographs</a></li>
+                    <li><a href="/museum/search/">Museum</a></li>
+                    <li><a href="/memorials/search/">Memorials</a></li>               
+                </ul>
+            </div>
         </div>
     </header>
     <div id="application-container" class="container"></div>
-    <footer class="navbar navbar-default navbar-fixed-bottom">
+    <footer class="navbar navbar-default navbar-static-bottom">
```

Comment:
As mentioned in chat (references are to a different project)...

[3:38:20 PM] Ben New: Worry about it in a future PR, but I just realised that the footer now sticks to the bottom of the content when the content is shorter than the page, what we really want is a footer that sticks to whichever is lower, i.e. the window if the content is short or the content if the content is longer
[3:38:47 PM] Ben New: e..g this page has short  content so the footer is pinned
[3:38:48 PM] Ben New: https://staging.wanma.org.au/editors
[3:39:01 PM] Ben New: This page has long content so the footer follows the content
[3:39:07 PM] Ben New: https://staging.wanma.org.au/people
[3:39:33 PM] Ben New: Probably needs a bit of javascript
[3:39:47 PM] Ben New: At least I used javascript in WANMA to get that effect